packages/sls-mentor/src/rules/noTimeout/index.ts

```diff
@@ -0,0 +1,30 @@
+import { FunctionConfiguration } from '@aws-sdk/client-lambda';
+import { fetchAllLambdaConfigurations } from '../../aws-sdk-helpers';
+import { SlsMentorLevel } from '../../constants/level';
+import { Category, Rule } from '../../types';
+
+const hasTimeout = (lambdaConfiguration: FunctionConfiguration) =>
+  lambdaConfiguration.Timeout !== undefined;
+
+const run: Rule['run'] = async resourceArns => {
+  const lambdaConfigurations = await fetchAllLambdaConfigurations(resourceArns);
+
+  const results = lambdaConfigurations.map(({ arn, configuration }) => ({
+    arn,
+    success: !hasTimeout(configuration),
```

Comment:
```suggestion
    success: hasTimeout(configuration),
```

It should  succeed if Lambda has a  timeout set, not sure I understand the presence of `!` here